Vizzuality/landgriffon
# Bugfix LANDGRIF-970 fix table chart grouping by

### General description

This PR fixes a query error exception thrown when grouping by Supplier AND adding any other filter, for all charts and tables

### Designs

_Link to the related design prototypes (if applicable)_

### Testing instructions

In any view of table or charts, group by supplier and add random filtering criteria. A table should be retrieved

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
